pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[tool.towncrier]
+package = "pytest"
+filename = "CHANGELOG.rst"
+directory = "news/"
```

Comment:
Small suggestion: since we are calling it `CHANGELOG`, perhaps this directory should be `changes/`? I don't feel strongly about it, just a thought.